include/NumCpp/Converters.hpp

```diff
@@ -0,0 +1,50 @@
+/// @file
+/// @author Levente Torok <toroklev@gmail.com>
+/// [GitHub Repository](https://github.com/dpilger26/NumCpp)
+///
+/// License
+/// Copyright 2018-2023 Levente Torok
+///
+/// Permission is hereby granted, free of charge, to any person obtaining a copy of this
+/// software and associated documentation files(the "Software"), to deal in the Software
+/// without restriction, including without limitation the rights to use, copy, modify,
+/// merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+/// permit persons to whom the Software is furnished to do so, subject to the following
+/// conditions :
+///
+/// The above copyright notice and this permission notice shall be included in all copies
+/// or substantial portions of the Software.
+///
+/// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+/// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+/// PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+/// FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+/// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+/// DEALINGS IN THE SOFTWARE.
+///
+/// Description
+/// Functions for working with NdArrays
+///
+#pragma once
+
+#include <initializer_list>
+#include <string>
+
+#include "NumCpp/Core/Internal/Error.hpp"
+#include "NumCpp/Core/Types.hpp"
+#include "NumCpp/NdArray.hpp"
+
+namespace nc
+{
+    template<typename dtype>
+    std::list<NdArray<dtype>> convert_initializer_list_2_list(std::initializer_list<NdArray<dtype>> inArrayList)
```

Comment:
Is there are reason you chose `std::list` as your return type rather than `std::vector`?  I think `std::vector` would be more appropriate for this.  I think if `std::vector` is just used throughout your other changes and your `initializer_list` overloads are removed, all functionality would still be there without the need to the conversion at all.

Reply:
This file needs to be moved to `include/Core/Internal`